GiniSwitchSDK/GiniSwitchSDK/Classes/GiniSwitchSdk.swift

```diff
@@ -24,13 +24,19 @@ public protocol GiniSwitchSdkDelegate: class {
 
 public class GiniSwitchSdk {
     
+    /// The client ID you got from Gini GmbH
     public let clientId: String
+    /// The client secret, securing your account, provided by Gini GmbH
     public let clientSecret: String
+    /// Your domain. Will be used when creating new users
     public let clientDomain: String
     
+    /// The delegate object, receiving callbacks about events happening in the SDK
     public weak var delegate:GiniSwitchSdkDelegate? = nil
     
+    /// The object responsible for creating the SDK's UI
     let userInterface = GiniSwitchUserInterface()
+    /// Contains some configuration objects that can be used to customize the SDK
```

Comment:
It's a short documentation comment in some documentation tools. I still don't know which one I'd use, but still...